src/components/PreviousExecTeam/PreviousExecTeam.tsx

```diff
@@ -0,0 +1,165 @@
+      }
+    }
+  };
+  const handleRightArrow = () => {
+    const selectedYearIndex = getSelectedYearIndex();
+    if (selectedYearIndex < prevTeamsData.length - 1) {
+      setSelectedYear(prevTeamsData[selectedYearIndex + 1].year);
+
+      // Recalculate visible years only if necessary
+      const currentStartIndex = prevTeamsData.findIndex((data) => data.year === visibleYears[0].year);
+      const numOfVisibleYears = props.desktopView ? VISIBLE_YEARS_DESKTOP : VISIBLE_YEARS_MOBILE;
+
+      // If the selected year moves beyond the currently visible years
+      if (selectedYearIndex + 1 >= currentStartIndex + numOfVisibleYears) {
+        // Shift the visible years by one to the right
+        const newStartIndex = Math.min(prevTeamsData.length - numOfVisibleYears, currentStartIndex + 1);
+        const newEndIndex = Math.min(prevTeamsData.length, newStartIndex + numOfVisibleYears);
+        setVisibleYears(prevTeamsData.slice(newStartIndex, newEndIndex));
+      }
+    }
+  };
+
+
+  return (
+    <S.PrevTeamSection>
+      <Divider />
+      <S.SectionHeader>Previous Executives</S.SectionHeader>
+      <S.PaginationControl>
+        <S.ArrowButton
+          className="arrow-button"
+          onClick={handleLeftArrow}
+          disabled={getSelectedYearIndex() === 0}
+        >
+        <FontAwesomeIcon icon={faChevronLeft} size="sm" style={{color:"black"}} />
+        </S.ArrowButton>
+        <S.YearPagination>
+          {visibleYears.map((data) => (
+            <S.YearButton
+              key={data.year}
+              className={selectedYear === data.year ? 'selected' : ''}
+              onClick={() => setSelectedYear(data.year)}
+            >
+              {data.year}
+            </S.YearButton>
+          ))}
+        </S.YearPagination>
+        <S.ArrowButton
+          className="arrow-button"
+          onClick={handleRightArrow}
+          disabled={getSelectedYearIndex() === prevTeamsData.length - 1}
+        >
+        <FontAwesomeIcon icon={faChevronRight} size="sm" style={{color:"black"}} />
+        </S.ArrowButton>
+      </S.PaginationControl>
+      
+      <S.Divider />
+      
+      <S.TeamList>
+        {prevTeamsData
+          .filter((team) => team.year === selectedYear)
+          .map((team) => {
+            const totalMember = team.members.length;
+            const membersPerRow = props.desktopView ? 2 : 1; // based on the view, how many columns
+            const lastRowStartIndex = Math.floor((totalMember-1)/membersPerRow)*membersPerRow; // index of element that will be on the last row
+            return(
+            <>
+              {team.members.length > 0 && (
+                <>
+                  {team.members.map((member, index) => (
+                    <S.TeamMember 
+                      key={index}
+                      lastRow={index>= lastRowStartIndex ? true : false} // if index is greater or equal, than we are on last row
```

Comment:
This file needs to be beautified, there's uneven spacing around the `>=`, just run Prettier on it.